lib/manageiq/api/common/rbac/acls.rb

```diff
@@ -0,0 +1,74 @@
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class ACLS
```

Comment:
Minor, but should this be `ACLs` (lower case s)`?

Reply:
Changed into `ACL`